### DISCUSSION TITLE
Suport for Micro Reciprocal Degree (Mired) to represent color temperature

### DISCUSSION THREAD

**Discussion by k3nsei** [angularsen/UnitsNet#1438] [Ideas]
It would be nice to have support for representing color temperature in mireds and be able to convert it to Kelvins. It's useful when working with light apis.

**Comment by angularsen** [comment]
Sounds useful, would you be interested in attempting a pull request?

Detailed steps here:
https://github.com/angularsen/UnitsNet/wiki/Adding-a-New-Unit

**Comment by k3nsei** [comment]
Normally I would but I'm newby to C# and DotNet. I'm mostly working with TypeScript.

**Reply by angularsen** [reply]
I see, well it will probably not get done until someone with the need for this in UnitsNet attempts a pull request. It is key to how we build UnitsNet, to ensure it will be used and its correctness is verified by someone working in the domain.
